internal/operator/cluster/upgrade.go

```diff
@@ -840,3 +847,65 @@ func publishUpgradeClusterFailureEvent(eventHeader events.EventHeader, clusterna
 		log.Errorf("error publishing event. Error: %s", err.Error())
 	}
 }
+
+// updatePGConfigDCS updates PG configuration for cluster via its Distributed Configuration Store
+// (DCS) according to the key/value pairs defined in the pgConfig map, specifically by updating
+// the <clusterName>-pgha-config ConfigMap.  Specifically, the configuration settings specified are
+// applied to the entire cluster via the DCS configuration included within this the
+// <clusterName>-pgha-config ConfigMap.
+// Note: Original version of this function from ./testing/pgo_cli/suite_helpers_test.go
+func updatePGConfigDCS(clientset kubernetes.Interface, clusterName, namespace string, pgConfig map[string]string) error {
+
+	dcsConfigName := fmt.Sprintf("%s-dcs-config", clusterName)
```

Comment:
We already have this defined:

https://github.com/CrunchyData/postgres-operator/blob/775fd1e8450fab5289e72a79617cc693f7f72c51/internal/operator/config/dcs.go#L39

Reply:
updated, thanks.